### DISCUSSION TITLE
<Your question or feedback about the GitHub Code Search and Code View products>

### DISCUSSION THREAD

**Discussion by ulision** [community/community#76007] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

bonjour , excusé moi par avance si je ne suis pas au bon endroit . y a tellement de rubriques,  je suis perdu.
j'ai deux questions : 
la première : comment trouver des sources , qui traite sur l'exportation d'une base SQLite local existante dans un dossier assets d'une application .je cherche à faire cela. mais , je ne trouve aucune source pour m'aiguiller , ni aide 
ma deuxième , je cherche toutes les sources qui utilise comme moi une base SQLite existante  dans le projet .
je suis débutant , jamais pris de cours et beaucoup trop vieux maintenant pour en prendre.
merci pour votre aide.

**Comment by Akash1134** [comment]
Hi @ulision , thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
